plugins/openapi/pkg/openapiclient/openapi.go

```diff
@@ -0,0 +1,181 @@
+package openapiclient
```

Comment:
I would rename the package to `openapi` and make the type `Client` so it is not repeatative.

`openapiclient.NewOpenAPIClient` is an undesirable stutter

openapi.New(..)
or 
openapi.NewClient(...)